CentralizedConsentAPI/README.md

```diff
@@ -0,0 +1,184 @@
+user to opt-in globally.
+
+### Default value
+
+The default value will be ‘No Preference’ to abide by some consumer privacy legislation
+which specifies that the user must indicate their privacy preference [affirmatively](https://oag.ca.gov/sites/all/files/agweb/pdfs/privacy/ccpa-fsor-appendix-e.pdf).
+The user agent will determine how they will prompt a user to set this value.
+Having a default value of ‘No Preference’ will also avoid adding more disincentive for
+sites to adopt.
+
+### Make per-site and global preferences indistinguishable
+
+The Centralized Consent API was designed to make it impossible to determine whether the
+result of `document.dataConsentPreference()` is returning a global or per-site value to avoid
+adding a global fingerprinting bit. It will be up to the user agent to determine how to store and prompt
+for global and per-site preferences.
+
+### Limiting consent information to two bits
+
+Two bits of information provides one bit for each current legislation requiring consent
+banners (CCPA and GDPR). It also could be used to hold bits of information regarding
+which types of cookies a user has opted into, outlined [here](https://gdprprivacypolicy.org/cookies-policy/).
+However, there is no way to enforce that the two bits be used for consent information,
+and it provides additional fingerprinting entropy.
+
+### Legal effects
+
+Currently, the Centralized Consent API is structured so that a site must query for a
+user’s data selling/sharing consent preference. In some jurisdictions, a
+user’s preference could act as a user-enabled global privacy control, and could make a site legally obligated to abide by that preference (see CCPA [regulations](https://www.oag.ca.gov/sites/all/files/agweb/pdfs/privacy/oal-sub-final-text-of-regs.pdf), section 999.315).
+
+
+## Stakeholder Signals
+[Global Privacy Control](https://globalprivacycontrol.github.io/gpc-spec/) (GPC) is a proposal to
+  standardize ‘Do Not Sell’. This work has some overlap with our purpose, but differs in
+  a few crucial ways.
+  * GPC actively pushes the Sec-GPC value to all sites via HTTP header, and is relying on legal enforcement under CCPA/GDPR to drive adoption. By contrast, the Centralized Consent API will rely on a site’s incentive to create a better user experience and have more persistent storage for consent data. It may use legal enforcement if applicable once a site obtains a user’s preference.
+  * The Centralized Consent API avoids adding a global fingerprinting bit by making per-site and global preferences indistinguishable, and mitigates fingerprinting risks further by only revealing a user’s preference in a first-party context with user interaction.
+
+
+## Acknowledgements
+
+Many thanks for valuable feedback and advice from other members of the WebKit team, including Tess, Brent, John and Simon.
+
+[CSS Pseudo-Elements]: https://drafts.csswg.org/css-pseudo-4/
```

Comment:
It looks like all of the lines from here to the end of the document are things accidentally copied from another explainer?